tutorials/how_to/elmo.md

```diff
@@ -53,10 +52,31 @@ JSON-serialized string with a mapping from sentences to line indices to the
 
 For more details, see `allennlp elmo -h`. 
 
-## Using ELMo programmatically
+## Using ELMo interactively
```

Comment:
Can you swap the order of these subsections?

Put "Using ELMo as a PyTorch `Module`" first, then `Using ELMo interactively`, then `Writing contextual representations to disk` last.

Reply:
I'm don't think that's best since people who are new to the space will want a second that comes later (and will probably get tripped up by what they don't know if it's buried)--but I made the change.